src/refactorings/convert-for-to-foreach/convert-for-to-foreach.test.ts

```diff
@@ -193,6 +316,26 @@ items.forEach(item => {
     }
   );
 
+  it("should call onMatch for a matching for-loop", async () => {
```

Comment:
Not happy that this test is quite complex in its implementation, but nothing else is preventing us from forgetting to use the `onMatch()` argument.

I'll figure out something better after I did some more I guess.